pkg/fs/fs.go

```diff
@@ -138,14 +137,15 @@ func copyDir(src, dst string, info os.FileInfo) error {
 		return err
 	}
 	// copy every source dir entry
-	entries, err := ioutil.ReadDir(src)
+	entries, err := os.ReadDir(src)
 	if err != nil {
 		return err
 	}
 	for _, entry := range entries {
 		entrySrc := filepath.Join(src, entry.Name())
 		entryDst := filepath.Join(dst, entry.Name())
-		if err := copy(entrySrc, entryDst, entry); err != nil {
+		fileInfo, _ := entry.Info()
```

Comment:
we shouldn't discard this error?

Reply:
```entries, err := os.ReadDir(src)```
In the above code , If the err is nil , Maybe deduction the ```entry.Info()``` will return normal code without err .

But you are right , we should consider the error side..
I will update it later...